selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -89,29 +68,13 @@ void Networking::attemptInitialization() {
     }
   )");
   main_layout->setCurrentWidget(wifiScreen);
-  ui_setup_complete = true;
-  wifi->requestScan();
-}
-
-void Networking::requestScan() {
-  if (!this->isVisible()) {
-    return;
-  }
-  if (!ui_setup_complete) {
-    attemptInitialization();
-    if (!ui_setup_complete) {
-      return;
-    }
-  }
-  wifi->requestScan();
 }
 
 void Networking::refresh() {
-  if (this->isVisible() || firstRefresh) {
+  if (wifi->seen_networks.size() != 0) {
```

Comment:
does this make a difference? refresh should be performant enough to not need this

Reply:
This was so we don't show a blank screen when there's no networks around, but we can instead add the "Scanning for networks" whenever there's no networks